server/graphql/v2/scalar/EmailAddress.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLError, GraphQLScalarType, Kind } from 'graphql';
+import { isEmail } from 'validator';
+
+const validate = (value: unknown) => {
+  if (typeof value !== 'string') {
+    throw new TypeError(`Value is not string: ${value}`);
+  }
+
+  if (!isEmail(value)) {
+    throw new TypeError(`Value is not a valid email address: ${value}`);
+  }
+
+  return value;
+};
+
+/**
+ * A field whose value conforms to the standard internet email address format as specified in RFC822: https://www.w3.org/Protocols/rfc822/.
+ * Inspired by https://github.com/Urigo/graphql-scalars/blob/master/src/scalars/EmailAddress.ts
+ */
+const EmailAddress = new GraphQLScalarType({
```

Comment:
Good idea! Will help us check emails elsewhere as well, not just with guest contributions.